src/js/pages/GranthIndex/util/sanitize-hash.ts

```diff
@@ -1 +1,5 @@
-export const sanitizeHash = (...args) => args.map(a => a.replace(/ /gi, '')).join('-');
+export const sanitizeHash = (...args) =>
+  args.map((a) => a.replace(/ /gi, '-')).join('_');
+
+export const desanitzeHash = (hash: string) =>
```

Comment:
Added this to get back the path of the last clicked list item. With the path available, we should be able to programatically open the list item and scroll to it